### PR TITLE
e2e-fix tests with new link

### PR DESCRIPTION
**Pull request by tanyatrayanava**

### CODE REVIEW

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts [hunk 1]
```diff
@@ -256,7 +250,7 @@ test
             //Verify that user can use Read More link for Bloom, Cuckoo, CMS, TDigest, TopK groups in Command Helper (RedisBloom module).
             await t.click(cliPage.readMoreButton);
             //Check new opened window page with the correct URL
-            await t.expect(getPageUrl()).contains(externalPageLinks[i]);
+            await t.expect(getPageUrl()).eql(externalPageLinks[i]);
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts:253]: Error message is missed

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts [hunk 2]
```diff
@@ -214,7 +208,7 @@ test
         //Verify that user can use Read More link for Gears group in Command Helper (RedisGears module)
         await t.click(cliPage.readMoreButton);
         //Check new opened window page with the correct URL
-        await t.expect(getPageUrl()).contains(externalPageLink);
+        await t.expect(getPageUrl()).eql(externalPageLink);
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts:211]: Error message is missed

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts [hunk 3]
```diff
@@ -190,7 +184,7 @@ test
             //Click on Read More link for selected command
             await t.click(cliPage.readMoreButton);
             //Check new opened window page with the correct URL
-            await t.expect(getPageUrl()).contains(externalPageLinks[i]);
+            await t.expect(getPageUrl()).eql(externalPageLinks[i]);
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts:187]: Error message is missed

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts [hunk 4]
```diff
@@ -107,19 +106,16 @@ test.skip
         //Click on Read More link for selected command
         await t.click(cliPage.readMoreButton);
         //Check new opened window page with the correct URL
-        await t.expect(getPageUrl()).contains(externalPageLink);
-        //Check that command info is displayed on the page
-        await t.expect(cliPage.cliReadMoreJSONCommandDocumentation().textContent).contains('JSON.SET');
+        await t.expect(getPageUrl()).eql(externalPageLink);
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts:109]: The error message is missed. Please go over the whole file

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts (hunk 1 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts]: added

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts (hunk 2 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts]: added

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts (hunk 3 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts]: added

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts (hunk 4 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts]: added

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**